layouts/partials/footer/mermaid.html

```diff
@@ -0,0 +1,15 @@
+{{- $defaultMermaid := .Site.Data.config.default.mermaid -}}
+{{- $mermaid := ( or .Site.Data.config.mermaid $defaultMermaid ) -}}
+
+{{- if eq .Params.mermaid true -}}
+
+<script src='{{- (or $mermaid.library.path $defaultMermaid.library.path) -}}' />
+
+<script>
+    mermaid.initialize({ 
+        theme: '{{- (or $mermaid.theme $defaultMermaid.theme) -}}',
+        startOnLoad: true
+    });
+</script>
```

Comment:
Please replace that with this:

```html
<script type='text/javascript'>
  mermaid.initialize(
    {{- $mermaid.config | jsonify | safeJS -}}
  );
</script>
```

This will use the `config` object in the `mermaid.json` file. `startOnLoad` is `true` by default, so no need to set it.